include/zephyr/net/phy.h

```diff
@@ -104,24 +103,25 @@ typedef void (*phy_callback_t)(const struct device *dev,
  */
 __subsystem struct ethphy_driver_api {
 	/** Get link state */
-	int (*get_link)(const struct device *dev,
-			struct phy_link_state *state);
+	int (*get_link)(const struct device *dev, struct phy_link_state *state);
 
 	/** Configure link */
-	int (*cfg_link)(const struct device *dev,
-			enum phy_link_speed adv_speeds);
+	int (*cfg_link)(const struct device *dev, enum phy_link_speed adv_speeds);
 
 	/** Set callback to be invoked when link state changes. */
-	int (*link_cb_set)(const struct device *dev, phy_callback_t cb,
-			   void *user_data);
+	int (*link_cb_set)(const struct device *dev, phy_callback_t cb, void *user_data);
 
 	/** Read PHY register */
-	int (*read)(const struct device *dev, uint16_t reg_addr,
-		    uint32_t *data);
+	int (*read)(const struct device *dev, uint16_t reg_addr, uint32_t *data);
 
 	/** Write PHY register */
-	int (*write)(const struct device *dev, uint16_t reg_addr,
-		     uint32_t data);
+	int (*write)(const struct device *dev, uint16_t reg_addr, uint32_t data);
+
+	/** Read PHY C45 register */
+	int (*read_c45)(const struct device *dev, uint8_t devad, uint16_t regad, uint16_t *data);
+
+	/** Write PHY C45 register */
+	int (*write_c45)(const struct device *dev, uint8_t devad, uint16_t regad, uint16_t data);
 };
```

Comment:
Here the `write_c45` and `read_c45` are only added as well as below its implementations.

IMHO it would be better to split this patch to two - one with only code refactoring (no functional changes) and the
second with implementation of C45 access functions.

Reply:
Sure will split them.